carbon-design-system/carbon-addons-iot-react
# fix(table): fix a11y issues on column selection button and toolbar tooltip

Closes #2593 

**Change List (commits, features, bugs, etc)**

- changes button role from "presentation" to "button" in column selection row
- adds `toolbarTooltipLabel` to i18n to add aria-label to table toolbar tooltips
- fixes incorrect "options" parameter passed to some tests

**Acceptance Test (how to verify the PR)**

- go to basic dumb table test
- turn on hasColumnSelection knob
- open in new window
- scan with IBM Equal Access Accessibility Checker
- There should only be 5 violations about Bypass blocks and content not being within a landmark element

**Regression Test (how to make sure this PR doesn't break old functionality)**

- column selection drag and drop and hiding still works as expected



**Things to look for during review**
